kitchen.yml

```diff
@@ -17,29 +17,40 @@ platforms:
       image: netmanagers/salt-2019.2-py3:debian-9
   - name: ubuntu-1804-2019-2-py3
     driver:
-      image: netmanagers/salt-2019.2-py3:ubuntu-1804
-  - name: centos-7-2019-2-py2
+      image: netmanagers/salt-2019.2-py3:ubuntu-18.04
+  - name: centos-7-2019-2-py3
     driver:
-      image: netmanagers/salt-2019.2-py2:centos-7
-  - name: fedora-29-2019-2-py2
+      image: netmanagers/salt-2019.2-py3:centos-7
+  - name: fedora-29-2019-2-py3
     driver:
-      image: netmanagers/salt-2019.2-py2:fedora-29
-
-  ## SALT 2018.3
-  - name: opensuse-423-2018-3-py2
+      image: netmanagers/salt-2019.2-py3:fedora-29
+  - name: opensuse-leap-15-2019-2-py3
     driver:
-      image: netmanagers/salt-2018.3-py2:opensuse-423
+      image: netmanagers/salt-2019.2-py3:opensuse-leap-15
       run_command: /usr/lib/systemd/systemd
-  - name: debian-8-2018-3-py2
+
+  ## SALT 2018.3
+  - name: debian-9-2018-3-py2
     driver:
-      image: netmanagers/salt-2018.3-py2:debian-8
+      image: netmanagers/salt-2018.3-py2:debian-9
   - name: ubuntu-1604-2018-3-py2
     driver:
-      image: netmanagers/salt-2018.3-py2:ubuntu-1604
-  - name: fedora-28-2018-3-py2
+      image: netmanagers/salt-2018.3-py2:ubuntu-16.04
+  - name: centos-7-2018-3-py2
     driver:
-      image: netmanagers/salt-2018.3-py2:fedora-28
-
+      image: netmanagers/salt-2018.3-py2:centos-7
+  - name: fedora-29-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:fedora-29
+  # TODO: Use this when fixed instead of `opensuse-leap-42`
+  # - name: opensuse-leap-15-2018-3-py2
+  #   driver:
+  #     image: netmanagers/salt-2018.3-py2:opensuse-leap-15
+  #     run_command: /usr/lib/systemd/systemd
```

Comment:
This can be re-enabled?

Reply:
See the [comment above](https://github.com/saltstack-formulas/template-formula/pull/101#discussion_r280443591).